### PR TITLE
Fix/dev7219 remove download refs from data dictionary

### PR DESCRIPTION
**Pull request by brett-varney**
**High level description:**

Remove references to "Download Center" from the Data Dictionary page

**Technical details:**
This ended up being more than conditionally hiding the sidebar, so I updated the file structure and combined/simplified where I could.
- updated sitemap & route
- removed dd refs from download page
- moved dd files from download folders to their own
- renamed folders & files to remove casing (just in case)
- updated & simplified css
- updated internal links

**JIRA Ticket:**
[DEV-7219](https://federal-spending-transparency.atlassian.net/browse/DEV-7219)

**Mockup:**
https://bahdigital.invisionapp.com/share/S6IAFK24YKC#/screens

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge (Windows only)
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [x] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations [React Testing Library](react-testing-library.md)

Reviewer(s):
- [x] Design review complete
- [ ] Code review complete

### CODE REVIEW

File: src/js/containers/router/RouterRoutes.js [hunk 1]
```diff
@@ -35,6 +34,7 @@ const AboutTheDataPage = React.lazy(() => import('components/aboutTheData/AboutT
 const AgencyDetailsPage = React.lazy(() => import('components/aboutTheData/AgencyDetailsPage').then((comp) => comp));
 const ErrorPage = React.lazy(() => import('components/errorPage/ErrorPage').then((comp) => comp));
 const SubmissionStatisticsDataSources = React.lazy(() => import('components/aboutTheData/DataSourcesAndMethodologiesPage').then((comp) => comp));
+const DataDictionaryPage = React.lazy(() => import('components/dataDictionary/dataDictionaryPage').then((comp) => comp));
```
Comment (MaxwellKendall) [inline src/js/containers/router/RouterRoutes.js:37]: this file needs to be upper case

`DataDictionaryPage`

File: tests/containers/dataDictionary/dataDictionaryContainer-test.jsx [hunk 2]
```diff
@@ -5,39 +5,37 @@
...
 import React from 'react';
 import { mount } from 'enzyme';
-import DataDictionaryContainer from 'containers/bulkDownload/dictionary/DataDictionaryContainer';
-import { mockDictionary } from '../mockData';
+import DataDictionaryContainer from 'containers/dataDictionary/dataDictionaryContainer';
 
-// mock the bulkDownload helper
-jest.mock('helpers/bulkDownloadHelper', () => require('../mockBulkDownloadHelper'));
+import { mockApiCall } from '../../testResources/mockApiHelper';
+import * as api from '../../../src/js/helpers/apiRequest';
```
Comment (MaxwellKendall) [inline tests/containers/dataDictionary/dataDictionaryContainer-test.jsx:11]: these imports should be absolute. 

We have rules in our webpack config to make `helpers` resolve to `~/src/js/`

File: tests/mockApi/responses/dataDictionary.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+export const mockDictionary = {
```
Comment (MaxwellKendall) [inline tests/mockApi/responses/dataDictionary.js:1]: @jonhill13 behold thy children

**Review by MaxwellKendall — CHANGES_REQUESTED** [review]
a few small changes!

File: tests/containers/dataDictionary/dataDictionaryContainer-test.jsx (hunk 2 above)
Comment (brett-varney) [reply tests/containers/dataDictionary/dataDictionaryContainer-test.jsx]: thanks, done

File: src/js/containers/router/RouterRoutes.js (hunk 1 above)
Comment (brett-varney) [reply src/js/containers/router/RouterRoutes.js]: done

File: src/js/apis/agencyV2.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /**
- * agencyV2APIs.js
+ * agencyV2.js
```
Comment (jonhill13) [inline src/js/apis/agencyV2.js:2]: Thank you sir!

Comment (brett-varney) [reply src/js/apis/agencyV2.js:2]: Ah, crap. I knew I was going to merge in dev accidentally...

**Comment by brett-varney** [comment]
got other branches mixed in

**Pull request closed without merging**